app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -86,8 +86,9 @@ class BookmarksToolbar extends ImmutableComponent {
       }
     }
     if (e.dataTransfer.files.length > 0) {
```

Comment:
there are limitations on what we can do with this api without breaking compatibility so rather than change what is contained in file, we added the extra data to items